lib/iris/experimental/animate.py

```diff
@@ -41,7 +41,11 @@ def animate(cube_iterator, plot_func, fig=None, **kwargs):
         object. See :meth:`iris.cube.Cube.slices`.
 
     * plot_func (:mod:`~iris.plot` or :mod:`~iris.quickplot` plot):
-        Plotting function used to animate.
+        Plotting function used to animate. Must accept the signature 
```

Comment:
Not your fault, but I think the line above is nonsensical, 
renders as "plot_func (**plot** or **quickplot** plot):".
I'd guess that should really be  ""**plot** or **quickplot** _function_" ?